recipes/ad3/recipe/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "ad3" %}
+{% set version = "2.5.3" %}
+{% set sha256 = "1a63e2e401a33467febc22ca10e91d7385c6813b2c3b44e96460360b76db147c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: ad3-{{ version }}.tar.gz
+  url: https://github.com/DerThorsten/AD3/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - eigen
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/ad3  # [unix]
+    - test -f ${PREFIX}/include/ad3/Factor.h  # [unix]
+    - if exist %LIBRARY_PREFIX%\include\ad3\Factor.h (exit 0) else (exit 1)  # [win]
```

Comment:
Any file checks for the libraries installed?

Reply:
I add these things when I am back from vacations, looking forward to get this in codaforge